### PR TITLE
feat(bigquery): implement create and drop table method

### PR DESCRIPTION
**Pull request by krzysztof-kwitt**

### CODE REVIEW

File: ibis/backends/bigquery/__init__.py [hunk 1]
```diff
@@ -435,6 +437,31 @@ def set_database(self, name):
...
+    def create_table(
+        self,
+        name: str,
+        obj: pd.DataFrame | ir.Table | None = None,
+        schema: ibis.Schema | None = None,
+        database: str | None = None,
+    ) -> None:
+        if obj is not None:
+            raise NotImplementedError(
+                "Parameter obj is not supported for create_table method in BigQuery backend"
```
Comment (cpcloud) [inline ibis/backends/bigquery/__init__.py:449]: This could be a potentially interesting follow-up!

File: ibis/backends/bigquery/client.py [hunk 2]
```diff
@@ -72,6 +72,23 @@ def bigquery_schema(table):
     return sch.schema(fields)
 
 
+def ibis_schema_to_bigquery_schema(schema: sch.Schema):
```
Comment (cpcloud) [inline ibis/backends/bigquery/client.py:75]: Hm, I thought we had this function somewhere already.

Comment (cpcloud) [reply ibis/backends/bigquery/client.py:75]: I guess not!

Comment (krzysztof-kwitt) [reply ibis/backends/bigquery/client.py:75]: We have a function that allows us to convert in the opposite direction only.

Comment (cpcloud) [reply ibis/backends/bigquery/client.py:75]: Yep. I'll be putting up a PR in the next few days to clean up all the to_x/from_x ibis types/schemas.

**Comment by cpcloud** [comment]
BigQuery tests look good:

```
cloud in 🌐 falcon in …/ibis on  creae-drop-table-bigquery is 📦 v4.0.0 via 🐍 v3.10.9 via ❄️  impure (ibis-3.10.9)
❯ pytest -m 'bigquery' -n auto --dist=loadgroup --snapshot-update --allow-snapshot-deletion -q --maxfail=10
bringing up nodes...
.................................x.............................x.xx..xxxx..xxxx.....xx.x..xx.xxx......xxx......x....x.xxxxx...x...x........xx.........x...x......x.....xxx...x...x...x.......x [ 17%]
...xx.....x.......xxx.......xx...........x........................x........................x......................................................xx.xx..xx..xxx..xx.xx..x...xx...xx.x.x..xx.. [ 35%]
.xxx....x.x.x.xxx..x..x....xx..x...........xxx......x...x............x..............x...........................x.x.......x........x.x.x....x.......x..x.x...x...x..x..x...x.......xx........x [ 53%]
.xx.x......x....x......x....x.....x....xx...xx..x.......x..x.......x.............xx..x.....xx......x.x....x.....s.x.xxx.x.x..x..x.....xx...x.x..x..xx...xx.........x..........x.........xx..x. [ 70%]
..x......x......xxxx............x....x.....x.x..............x................................................................................................................................. [ 88%]
.....................x........x...............xx..x..xxx.................x............xs......................................                                                                 [100%]
891 passed, 2 skipped, 183 xfailed in 80.09s (0:01:20)
```

**Review by cpcloud — APPROVED** [review]

**Pull request merged**